packages/frontend-v2/utils/handleApiClientError.ts

```diff
@@ -40,7 +40,31 @@ const handleAxiosError = (error: AxiosError, router: NextRouter) => {
 
   if (statusCode === 400) {
     logger.debug("handleApiClientError", "Bad Request", error);
-    toast.error("Sorry, we sent some invalid data. Try again.");
+    let toastMsg = ""
+    const errorMsg = error.response?.data.message;
+
+    switch (errorMsg) {
+      case emailAlreadyExistsError:
+        toastMsg = 'The email you submitted already exists.'
+        break;
+      case emailDoesNotExistError:
+        toastMsg = "The email you entered does not exist. Please try again."
+        break;
+      case emailHasNotBeenConfirmed:
+        toastMsg = "The given email has not been confirmed yet. We can't reset your password."
+        break;
+      case weakPasswordError:
+        toastMsg = "The entered password is too weak"
+        break;
+      case wrongPasswordError:
+        toastMsg = "Incorrect Password!"
+        break;
+      default:
+        toastMsg = `Sorry, we sent some invalid data. ${errorMsg}`
+        break;
+    }
+
+    toast.error(toastMsg)
```

Comment:
oooh, yea I don't think we want this in the global error handling since this is all specific to the forgot/reset password endpoints. in the future we can handle endpoint specific error in the catch block and then delegate to handleApiClient error if it is something generic.